auth/uauth.go

```diff
@@ -48,7 +48,7 @@ var (
 // CreateAuthInfo interface provides necessary data
 type CreateAuthInfo interface {
 	Config() config.Provider
-	Logger() ulog.Log
+	Logger() *zap.Logger
```

Comment:
Let's discuss this in person. I Have a lot of reservations about top-level frameworks own interfaces for logging and metrics; it forces components far down the stack to depend on the framework, instead of the reverse.